src/mod.ts

```diff
@@ -139,6 +143,12 @@ class PathToTarkov implements IPreSptLoadMod, IPostSptLoadMod {
 
   public postSptLoad(container: DependencyContainer): void {
     this.container = container;
+    this.instanceManager.postDBLoad(container);
+
+    if (!this.config.enabled) {
+      return;
+    }
+
```

Comment:
This would break the current uninstall procedure.

This check is already done few lines later.